tensorflow_serving/model_servers/http_server.cc

```diff
@@ -169,6 +174,12 @@ class RestApiRequestDispatcher {
               << " " << req->uri_path() << " Error: " << status.ToString();
     }
     req->ReplyWithStatus(http_status);
+
```

Comment:
will this be better/readable:

```
if (http_status == net_http::HTTPStatusCode::OK) {
   RecordRequestLatency(model_name, /*api=*/method, /*entrypoint=*/"REST",
                                           Env::Default()->NowMicros() - start);
} else {
   VLOG(1) << "Error Processing HTTP/REST request: " << req->http_method()
                  << " " << req->uri_path() << " Error: " << status.ToString();
}
RecordModelRequestCount(model_name, status);
req->ReplyWithStatus(http_status);
```

so we do not time any socket write delay, the way we do not account for the read side when we get the request, and start our timing when we being processing the request above. this makes it symmetric.